views/index.hbs

```diff
@@ -6,6 +6,7 @@
   <meta name="description" content="{{description}}">
   <meta id="viewport" name="viewport" content="initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  {{contentPolicySecurity}}
```

Comment:
No, this is still sending it via a `<meta>` tag.  It needs to be sent as an HTTP header.

Reply:
Why is the meta tag not good ?